modules/administration/pages/ansible-integration.adoc

```diff
@@ -9,6 +9,7 @@ This feature is a technology preview.
 [NOTE]
 ====
 Currently, the supported version of Ansible is 2.9 (LTS) shipped in SUSE Linux Enterprise 15 Client Tools channels.
+The supported client version is SLES 15 SP3, or later.
```

Comment:
Here we mean SUMA clients, but to prevent confusion with Ansible clients, we can add something like this:

```suggestion
The supported OS version for the Ansible Control Node is SLES 15 SP3, or later.
```

Reply:
thank you for clarification, Can!